src/components/sections/people/PersonViewsPane.jsx

```diff
@@ -151,7 +164,9 @@ export default class PersonViewsPane extends RootPaneBase {
                             value={ this.state.query } objects={ queries } showEditLink={ true }
                             onValueChange={ (name, val) => this.setState({ query: val }) }
                             onCreate={ this.onQueryCreate.bind(this) }
-                            onEdit={ this.onQueryEdit.bind(this) }/>
+                            onEdit={ this.onQueryEdit.bind(this) }
+                            hidden={ this.state.collapseHeader }
```

Comment:
I would have preferred removing the `RelSelectInput` entirely instead of just hiding it with CSS. But that's a matter of taste, and soon enough we don't have to worry about things like that and instead let the UI library decide for us. For now, lets keep it like this. 😊